### PR TITLE
Reorder tabs

### PR DESCRIPTION
**Pull request by eshaben**
reorder network tabs!

1. Moonbeam
2. Moonriver
3. Moonbase Alpha
4. Moonbeam Dev Node

this PR also fixes some broken image paths

### CODE REVIEW

File: builders/interact/hardhat.md [hunk 1]
```diff
@@ -117,31 +117,32 @@ npm install @nomiclabs/hardhat-ethers ethers
...
 === "Moonbase Alpha"
     ```
     moonbase: {
         url: '{{ networks.moonbase.rpc_url }}',
         chainId: {{ networks.moonbase.chain_id }},  // {{ networks.moonbase.hex_chain_id }} in hex,
         accounts: [privateKeyMoonbase] // Insert your private key here
       },
+    ```   
+    
+=== "Moonbeam Dev Node"
```
Comment (albertov19) [inline builders/interact/hardhat.md:138]: Why do we have this here, but on other instances we had `Moonbeam开发节点` @hyd628

File: builders/interact/truffle.md [hunk 2]
```diff
@@ -119,14 +118,14 @@ module.exports = {
     },
     ```
 
-=== "Moonriver"
+=== "Moonbeam Dev Node"
```
Comment (albertov19) [inline builders/interact/truffle.md:121]: Here as well  @hyd628

File: node-operators/networks/tracing-node.md [hunk 3]
```diff
@@ -101,23 +106,18 @@ Geth的`debug`和`txpool` API以及OpenEthereum的`trace`模块提供一个非
     --name="Moonbeam-Tutorial (Embedded Relay)"
     ```
 
-=== "Moonriver"
+=== "Moonbeam Dev Node"
```
Comment (albertov19) [inline node-operators/networks/tracing-node.md:109]: Another one @hyd628

File: tokens/connect/metamask.md [hunk 4]
```diff
@@ -85,49 +85,46 @@ description: 通过此教程学习如何将MetaMask，一个以浏览器为基
...
     - Block Explorer (Optional): `{{ networks.moonbase.block_explorer }}`
 
-=== "Moonriver"
-
-    - Network Name: `Moonriver`
-    - RPC URL: `{{ networks.moonriver.rpc_url }}`
-    - ChainID: `{{ networks.moonriver.chain_id }}` (hex: `{{ networks.moonriver.hex_chain_id }}`)
-    - Symbol (Optional): `MOVR`
-    - Block Explorer (Optional): `{{ networks.moonriver.block_explorer }}`
+=== "Moonbeam Dev Node"
```
Comment (albertov19) [inline tokens/connect/metamask.md:102]: Another one @hyd628

File: tokens/connect/nifty.md [hunk 5]
```diff
@@ -37,22 +37,19 @@ description: 此教程将会带您了解如何连接Nifty钱包，一个基于
...
 
 === "Moonbase Alpha"
-
     ```
       {{ networks.moonbase.rpc_url }}
     ```
 
-=== "Moonriver"
-
+=== "Moonbeam Dev Node"
```
Comment (albertov19) [inline tokens/connect/nifty.md:50]: Another one @hyd628

**Review by albertov19 — CHANGES_REQUESTED** [review]
@hyd628 please look at the comments I made. All aside, good!

File: builders/interact/hardhat.md (hunk 1 above)
Comment (hyd628) [reply builders/interact/hardhat.md:138]: they are inconsistent. prob different people translated them. Moonbeam开发节点 is prob the better one to use here.

**Comment by albertov19** [comment]
@eshaben we need to change `Moonbeam Dev Node` for `Moonbeam开发节点` - besides that, all good

**Review by albertov19 — APPROVED** [review]

**Pull request merged**